lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -44,31 +52,44 @@ function PropCore.ValidSpawn(ply, model, isVehicle)
 	playerMeta.LimitHit = limithit
 	return ret
 end
+local ValidSpawn = PropCore.ValidSpawn
 
 local canHaveInvalidPhysics = {
 	delete=true, parent=true, deparent=true, solid=true,
 	shadow=true, draw=true, use=true, pos=true, ang=true,
-	manipulate=true
+	manipulate=true, [0] = true
 }
-function PropCore.ValidAction(self, entity, cmd)
-	if(cmd=="spawn" or cmd=="Tdelete") then return true end
-	if(!IsValid(entity)) then return self:throw("Invalid entity!", false) end
-	if(!canHaveInvalidPhysics[cmd] and !validPhysics(entity)) then return self:throw("Invalid physics object!", false) end
-	if(!isOwner(self, entity)) then return self:throw("You do not own this entity!", false) end
+
+function PropCore.ValidAction(self, entity, cmd, bone, index)
+	if cmd == "spawn" or cmd == "Tdelete" then return true end
+	if not IsValid(entity) then return self:throw("Invalid entity!", false) end
+	if not canHaveInvalidPhysics[cmd or 0] and not validPhysics(entity) then return self:throw("Invalid physics object!", false) end
```

Comment:
Can this be moved in the `if cmd then` block instead of using `or 0`. Can also remove the 0 from the table above.